.envrc

```diff
@@ -1,3 +0,0 @@
-if has nix; then
```

Comment:
if shell.nix is deleted I think this file can be deleted also?

Reply:
File deleted